layers/meta-balena-beaglebone/recipes-core/images/balena-image.bbappend

```diff
@@ -6,3 +6,14 @@ IMAGE_ROOTFS_SIZE:beaglebone-ai64 = "733184"
 IMAGE_INSTALL:append:beaglebone = " bb-org-overlays fix-mmc-bbb bb-wl18xx-bluetooth bb-wl18xx-wlan0"
 IMAGE_INSTALL:append:beaglebone-ai64 = " mmc-utils vxd-dec-fw"
 IMAGE_INSTALL:append:beaglebone-pocket = " boot-scripts"
+IMAGE_INSTALL:append:beagleplay = " mmc-utils"
+IMAGE_INSTALL:append:beagleplay-k3r5 = " mmc-utils"
+
+IMAGE_INSTALL:remove:beagleplay = " bb-wl18xx-btwl"
+IMAGE_INSTALL:remove:beagleplay-k3-r5 = " bb-wl18xx-btwl"
+
+IMAGE_ROOTFS_MAXSIZE:beagleplay = "1966080"
```

Comment:
If you want to increase the rootfs size you should only use IMAGE_ROOTFS_SIZE so you can delete this from here

Reply:
OK. Will rebuild and retest.

Reply:
Done.